styles.css

```diff
@@ -4,18 +4,15 @@
 }
 
 body {
-  display: grid;
-  justify-content: center;
   margin: 0;
   padding: 0;
 }
 
 .main-container {
   display: grid;
```

Comment:
- [ ] [OPTIONAL] You're not using the grid as much, kindly consider using it more to layout your page